### ISSUE TITLE
saveMissing not working with version 1.7.8 but work with 1.7.4

### ISSUE THREAD

**Issue by chesnel** [i18next/i18next-node#165]
Hello,
I made a basic test with saveMissing and unfortunately is not working with the last version 1.7.8 in npm. I downgrade the version like this npm install i18next@1.7.4 and now my example works perfectly with the option saveMissing : true.

Is that something changed with the 1.7.8 version and I do not declare it right my options?

```javascript
var i18n = require('i18next');

var option = { 
  resSetPath: 'locales/__lng__/__ns__.json',
  resGetPath : 'locales/__lng__/__ns__.json',
  saveMissing : true,
  lng : 'en',
  preload : ['en'],
  fallbackLng : 'en',
  debug :true,
  sendMissingTo: 'all'
};
 
i18n.init(option, function(){
  console.log(i18n.t('patate douce'));
});
```

thank you in advance

**Comment by jamuhl** [comment]
your configuration is correct. there's a bug i introduced on 1.7.8. will be fixed in https://github.com/i18next/i18next-node/commit/2fcc5ee79346e688b19d13089af6360d55ff02f6

will publish an updated version asap.

**Comment by jamuhl** [comment]
fixed in published v1.7.9
